README.md

```diff
@@ -204,30 +129,18 @@ Example:
 ## 👥 Authors <a name="authors"></a>
 
 > Mention all of the collaborators of this project.
+> 👤 **Author1**
 
-👤 **Author1**
-
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
-
-👤 **Author2**
-
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+- GitHub: @Chandan-devs-tech
+- Twitter: @ChandanGuptaDev
+- LinkedIn: www.linkedin.com/in/chandangupta-devs
+- Slack: @Chandan Gupta
```

Comment:
- [OPTIONAL] While you add all the required information to the README file :1st_place_medal:, I think it would be better if the author links have the link to your corresponding social network :+1: